### PR TITLE
Release v1.3.3

### PR DESCRIPTION
**Pull request by coollog**
Release on tag: https://github.com/GoogleCloudPlatform/app-gradle-plugin/releases/tag/v1.3.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## 1.3.3
```
Comment (loosebazooka) [inline CHANGELOG.md]: Also these changes.

https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/162
https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/156

Comment (coollog) [reply CHANGELOG.md]: Okay just added these!

**Comment by loosebazooka** [comment]
you can add the label `kokoro:run` to force kokoro to run.

**Comment by patflynn** [comment]
best thing to do is update the changelog in the PRs where you do the changes.

**Pull request merged**